engine/execution/ingestion/uploader/retryable_uploader_wrapper.go

```diff
@@ -174,32 +175,26 @@ func (b *BadgerRetryableUploaderWrapper) reconstructComputationResult(
 	// Get EDID from ExecutionResult in BadgerDB
 	executionResult, err := b.results.ByBlockID(blockID)
 	if err != nil {
-		log.Error().Err(err).Msgf(
-			"failed to retrieve ExecutionResult from Badger with BlockID %s", blockID.String())
-		return nil, err
+		return nil, fmt.Errorf("failed to retrieve ExecutionResult with BlockID %s. %w", blockID.String(), err)
 	}
 	executionDataID := executionResult.ExecutionDataID
 
 	// retrieving BlockExecutionData from EDS
 	executionData, err := b.execDataDownloader.Get(b.unit.Ctx(), executionDataID)
 	if executionData == nil || err != nil {
-		log.Error().Err(err).Msgf(
```

Comment:
Do we still want to log the error somewhere outside of this function, or does that already happen?

Reply:
The error will popup and be [logged here](https://github.com/onflow/flow-go/blob/leo/check-events-hash-when-block-has-no-events/engine/execution/ingestion/engine.go#L110). Also this uploader is disabled on mainnet